### PR TITLE
Increment zero-based column for javascript-eslint

### LINKED ISSUE #639: Zero-based columns in javascript-eslint checker
Flycheck version: 0.24alpha (package: 20150503.1047)
eslint version v0.20.0

I believe there is a `(flycheck-increment-error-columns errors)` missing in the error-filter of the javascript-eslint checker.

This change works for me:
```diff
- (flycheck-sanitize-errors errors))
+ (flycheck-sanitize-errors (flycheck-increment-error-columns errors)))
```

Do you want me to make a pull request with that change?

### PR DESCRIPTION
**Pull request by hekto**
This should fix #639. I updated the test cases accordingly.

### CODE REVIEW

File: test/resources/checkers/javascript-syntax-error.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-/** A missing semicolon */
+/** A bad if */
 
-var foo = "Hello world);
+if ( /* nothing here */ ) // comment
```
Comment (lunaryorn) [inline test/resources/checkers/javascript-syntax-error.js]: Just curious, but why did you change the test case?

**Comment by lunaryorn** [comment]
@hekto Thanks.

File: test/resources/checkers/javascript-syntax-error.js (hunk 1 above)
Comment (hekto) [inline test/resources/checkers/javascript-syntax-error.js]: I didn't want the error to be on the last character of the line (or beyond?)... Not sure if it mattered, but it was easier to verify this way.

Comment (lunaryorn) [inline test/resources/checkers/javascript-syntax-error.js]: Ah, ok.  Good change then, thanks!

**Comment by lunaryorn** [comment]
@hekto Merged :)

**Comment by hekto** [comment]
Seems like this needs to be undone :)
http://eslint.org/docs/user-guide/migrating-to-1.0.0#column-numbers-are-1-based

**Comment by lunaryorn** [comment]
@hekto Is eslint 1.0 a thing already? I mean, this PR was introduced just two months ago?

**Comment by lunaryorn** [comment]
@hekto If 1.0 is already out, please open a new issue, or ideally a PR that reverts this one…

**Comment by hekto** [comment]
I'll have a look!
